k9mail-library/src/main/java/com/fsck/k9/mail/internet/FlowedMessageUtils.java

```diff
@@ -0,0 +1,20 @@
+package com.fsck.k9.mail.internet;
+
+
+import static com.fsck.k9.mail.internet.MimeUtility.getHeaderParameter;
+import static com.fsck.k9.mail.internet.MimeUtility.isFormatFlowed;
+
+
+public class FlowedMessageUtils {
```

Comment:
The method was in `MimeUtility` before and I asked for it to be moved. I didn't realize `isFormatFlowed()` was in there as well. You're right that these should stay together. However, I don't like that we just throw everything into `MimeUtility`. This is specialized functionality, not generic "MIME" functionality.
Those two methods could easily live inside the original `FlowedMessageUtils`. But for some reason `MessageExtractor` is part of k9mail-library, not the k9mail module. `FlowedMessageUtils`, on the other hand, lives in the app module.